lib/susedistribution.pm

```diff
@@ -134,7 +134,8 @@ sub ensure_installed {
     testapi::assert_script_sudo("chown $testapi::username /dev/$testapi::serialdev");
     my $retries = 5;    # arbitrary
     $self->script_run("for i in {1..$retries} ; do pkcon install $pkglist && break ; done ; RET=\$?; echo \"\n  pkcon finished\n\"; echo \"pkcon-\${RET}-\" > /dev/$testapi::serialdev", 0);
-    my @tags = qw/Policykit Policykit-behind-window pkcon-proceed-prompt pkcon-finished/;
+    my @tags        = qw/Policykit Policykit-behind-window pkcon-proceed-prompt pkcon-finished/;
```

Comment:
not yet, only after my after PR is merged. I don't like to change the style on single lines when the focus should be on something else, in this case that I changed other parts of the code.